requirements.txt

```diff
@@ -1,6 +1,6 @@
 celery >= 4.1.1, < 4.1.999
 docopt >= 0.6.2, < 0.6.999
-flask >= 0.10.1, < 0.10.999
+flask >= 0.12.3, < 0.12.999
```

Comment:
Don't you want the first version to be 0.12.4?

Reply:
0.12.3 is enough for the CVE issue: https://nvd.nist.gov/vuln/detail/CVE-2018-1000656